### PR TITLE
Add calendar view

### PR DESCRIPTION
**Pull request by aganpule**

### CODE REVIEW

File: .idea/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...eratorsGroup><Generator name="active_record:migration" /><Generator name="active_record:model" /><Generator name="active_record:observer" /><Generator name="active_record:session_migration" /><Generator name="controller" /><Generator name="erb:controller" /><Generator name="erb:mailer" /><Generator name="erb:scaffold" /><Generator name="generator" /><Generator name="helper" /><Generator name="integration_test" /><Generator name="mailer" /><Generator name="metal" /><Generator name="migration" /><Generator name="model" /><Generator name="model_subclass" /><Generator name="observer" /><Generator name="performance_test" /><Generator name="plugin" /><Generator name="resource" /><Generator name="scaffold" /><Generator name="scaffold_controller" /><Generator name="session_migration" /><Generator name="stylesheets" /><Generator name="test_unit:controller" /><Generator name="test_unit:helper" /><Generator name="test_unit:integration" /><Generator name="test_unit:mailer" /><Generator name="test_unit:model" /><Generator name="test_unit:observer" /><Generator name="test_unit:performance" /><Generator name="test_unit:plugin" /><Generator name="test_unit:scaffold" /></GeneratorsGroup></Settings>
```
Comment (jeremyjpark) [inline .idea/.generators]: Should these be in the repository?

File: .idea/.rakeTasks [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Settings><!--This file was automatically generated by Ruby plugin.
+You are allowed to: 
+1. Remove rake task
+2. Add existing rake tasks
+To add existing rake tasks automatically delete this file and reload the project.
+--><RakeGroup description="" fullCmd="" taksId="rake" /></Settings>
```
Comment (jeremyjpark) [inline .idea/.rakeTasks]: This too?

File: .idea/Datong.iml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    <orderEntry type="library" scope="PROVIDED" name="sprockets-rails (v3.2.0, rbenv: 2.3.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="thor (v0.19.1, rbenv: 2.3.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="thread_safe (v0.3.5, rbenv: 2.3.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="tzinfo (v1.2.2, rbenv: 2.3.1) [gem]" level="application" />
+  </component>
+  <component name="RModuleSettingsStorage">
+    <LOAD_PATH number="0" />
+    <I18N_FOLDERS number="1" string0="$MODULE_DIR$/config/locales" />
+  </component>
+</module>
```
Comment (jeremyjpark) [inline .idea/Datong.iml]: And these?

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <OptionsSetting value="true" id="Remove" />
+    <OptionsSetting value="true" id="Checkout" />
+    <OptionsSetting value="true" id="Update" />
+    <OptionsSetting value="true" id="Status" />
+    <OptionsSetting value="true" id="Edit" />
+    <ConfirmationsSetting value="0" id="Add" />
+    <ConfirmationsSetting value="0" id="Remove" />
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="rbenv: 2.3.1" project-jdk-type="RUBY_SDK" />
+</project>
```
Comment (jeremyjpark) [inline .idea/misc.xml]: This directory is machine specific to your installation of rubymine.

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Datong.iml" filepath="$PROJECT_DIR$/.idea/Datong.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (jeremyjpark) [inline .idea/modules.xml]: This one too

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,833 @@
...
+    <entry file="file://$PROJECT_DIR$/db/schema.rb">
+      <provider selected="true" editor-type-id="text-editor">
+        <state relative-caret-position="247">
+          <caret line="48" column="0" selection-start-line="48" selection-start-column="0" selection-end-line="48" selection-end-column="0" />
+          <folding />
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (jeremyjpark) [inline .idea/workspace.xml]: I think this is the last one

File: app/assets/javascripts/application.js [hunk 7]
```diff
@@ -14,12 +14,19 @@
...
+//= require_tree
+//= require moment 
+//= require fullcalendar
+
 
 $( document ).ready(function(){
     $(".button-collapse").sideNav();
 });
 
+// $( document ).ready(function(){
```
Comment (jeremyjpark) [inline app/assets/javascripts/application.js]: Do we still need this commented out portion?

File: app/assets/stylesheets/events.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Events controller here.
```
Comment (jeremyjpark) [inline app/assets/stylesheets/events.scss]: Do we need this file if it's essentially blank?

File: app/helpers/events_helper.rb [hunk 9]
```diff
@@ -0,0 +1,2 @@
+module EventsHelper
```
Comment (jeremyjpark) [inline app/helpers/events_helper.rb]: More or less an empty file

File: config/routes.rb [hunk 10]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :events
```
Comment (jeremyjpark) [inline config/routes.rb]: You might want to put this lower as rails routing is done from top to bottom. I don't think it's strictly necessary, but it's something to keep in mind.

File: features/event_calendar.feature [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+    And I press "Create Event"
+
+  Scenario: visiting the events page
+    Given I am on the events page
+    Then I should see Upcoming Events
+    And the page should have a div#calendar element
+
+  Scenario: Creating a new event
+    Given I am on the events page
+  #   And I am an admin
```
Comment (jeremyjpark) [inline features/event_calendar.feature]: Is this line required for this pull request?

File: features/step_definitions/web_steps.rb [hunk 12]
```diff
@@ -45,6 +45,17 @@ def with_scope(locator)
   visit path_to(page_name)
 end
 
+
+Given /^I\'m on the edit page for (.*)$/ do |event_title|
+  event = Event.find_by(title: event_title)
+  visit edit_event_path(event)
+
+  #event = Event.where(title: event_title)
```
Comment (jeremyjpark) [inline features/step_definitions/web_steps.rb]: I left these commented but just wanted to make sure that you took a look at them and understood the difference between ActiveRecord::where and ActiveRecord::find_by.

These can be removed as well.

**Review by LizzySteger — APPROVED** [review]
Looks good to me!

**Review by jeremyjpark — CHANGES_REQUESTED** [review]
Left comments on your PR.

**Comment by aganpule** [comment]
Resolved all the code review comments from @jeremyjpark

**Review by jeremyjpark — APPROVED** [review]

**Comment by jeremyjpark** [comment]
LGTM

**Pull request merged**